models.topic/topic.model.js

```diff
@@ -0,0 +1,10 @@
+const fs = require('fs/promises');
+const db = require('../db/connection');
+
+const readTopics = ()=> {
+    return db.query('SELECT * FROM topics').then((rows) => {
+        return rows
+    })
+};
+
```

Comment:
Rows here is not destructured, if you destructure it you can access res.body.topics in your tests instead of res.body.topics.rows.